Sources/VLCVideoOptionsControlBar.swift

```diff
@@ -0,0 +1,142 @@
+/*****************************************************************************
+ * VLCMediaViewEditCell.swift
+ *
+ * Copyright © 2018 VLC authors and VideoLAN
+ * Copyright © 2018 Videolabs
+ *
+ * Authors: Robert Gordon <robwaynegordon@gmail.com>
+ *
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+@objc protocol VLCVideoOptionsControlBarDelegate: class {
+    func didToggleFullScreen(_ optionsBar: VLCVideoOptionsControlBar)
```

Comment:
I believe the protocol name in Swift shouldn't have the VLC prefix like so: `VideoOptionsControlBarDelegate`.
Though, when expositing it to Objective-C, we should add the prefix.
Therefore we should have something like this:
```
@objc(VLCVideoOptionsControlBarDelegate)
protocol VideoOptionsControlBarDelegate...
```

Same remark for the filename.